configurator/main.c

```diff
@@ -8,8 +8,7 @@
 #include "usb.h"
 
 void get_all(void) {
-  int value;
-  printf("            TUNING\n");
```

Comment:
What did this heading text do to offend you?

Reply:
argl, nothing. that's VSCode's "great" staging of "single" lines.